browser.js

```diff
@@ -4,15 +4,17 @@ var readystate = module.exports = require('./readystate')
   , win = (new Function('return this'))()
   , complete = 'complete'
   , root = true
-  , doc = win.document
-  , html = doc.documentElement;
+  , doc = win ? win.document : null
+  , html = doc ? doc.documentElement : null;
 
 (function wrapper() {
   //
   // Bail out early if the document is already fully loaded. This means that this
   // script is loaded after the onload event.
   //
-  if (complete === doc.readyState) {
+  var docReadyState = function () { return doc ? doc.readyState : null; }
```

Comment:
Can we move this at the top level? Outside of the wrapper function.

Reply:
done